src/pages/ImportWorkflow/ImportWorkflow.test.ts

```diff
@@ -1,10 +1,11 @@
-import { act, render, screen } from '@testing-library/react';
```

Comment:
There were lots of unnecessary `act` calls in this test, so I removed them while I was here.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily